src/index.js

```diff
@@ -4,27 +4,42 @@ import DuplicatesInd1 from './duplicates-ind1';
 import EmptyFields from './empty-fields';
 import EndingPunctuation from './ending-punctuation';
 import EndingWhitespace from './ending-whitespace';
+import Field505Separators from './field-505-separators';
 import Field521Fix from './field-521-fix';
-import FieldsPresent from './fields-present';
+import FieldExclusion from './field-exclusion';
 import FieldStructure from './field-structure';
+import FieldsPresent from './fields-present';
 import FixedFields from './fixed-fields';
-import FieldExclusion from './field-exclusion';
+import FixRelatorTerms from './fixRelatorTerms';
 import IdenticalFields from './identical-fields';
+import IndicatorFixes from './indicator-fixes';
 import IsbnIssn from './isbn-issn';
 import ItemLanguage from './item-language';
+import MergeField500Lisapainokset from './mergeField500Lisapainokset';
+import MergeRelatorTermFields from './mergeRelatorTermFields';
+import MultipleSubfield0s from './multiple-subfield-0';
 import NonBreakingSpace from './non-breaking-space';
+import NormalizeIdentifiers from './normalize-identifiers';
+import NormalizeQualifyingInformation from './normalize-qualifying-information';
 import NormalizeUTF8Diacritics from './normalize-utf8-diacritics';
 import Punctuation from './punctuation/';
-import ResolveOrphanedSubfield6s from './resolveOrphanedSubfield6s'; // Do this before reindexing!
+import Punctuation2 from './punctuation2';
+import ResolveOrphanedSubfield6s from './resolveOrphanedSubfield6s'; // Do this before reindexing! (thus not in alphabetical order)
```

Comment:
I don't think the order of import command is meant in "Do this before reindexing"